2015/index.html

```diff
@@ -35,20 +35,22 @@
   </head>
   <body>
     <div class="wrapper">
-      <header>
-        <h1 class="title"><a href="/"><span class="title__colour">Tech</span>Week</a></h1>
-        <p>Dublin City University 9<sup>th</sup> - 13<sup>th</sup> March 2015</p>
-        <ul class="menu">
-          <li class="menu__item">
-            <a href="../2016">Feb 2016</a>
-          </li>
-          <li>
-            <a href="..">Oct 2016</a>
-          </li>
-        </ul>
-      </header>
-      <div class="container playlist">
-        <iframe id="live" src="https://www.youtube.com/embed/videoseries?list=PLPaLJSHTDMFgYNnnEOjuvdm6WqOFTSQ9z" width="100%" height="400" marginwidth="0" marginheight="0" scrolling="no" frameborder="0" webkitallowfullscreen="" allowfullscreen=""></iframe>
+      <div class="main">
```

Comment:
this isnt the main content of the page the main content would be the video and evens cards